reactnative/ReactNativeTemplateApp/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ReactNativeTemplateApp",
+  "name": "ReactNativeTemplate",
```

Comment:
The value in MainActivity has to match the registerComponent call in index.android.js (last line) and they do.
The name in the package.json really doesn't matter.